api/helpers/game-states/moves/stalemate-accept/execute.js

```diff
@@ -0,0 +1,52 @@
+const GamePhase = require('../../../../../utils/GamePhase.json');
+
+module.exports = {
+  friendlyName: 'Accept offer for Stalemate',
+
+  description: 'Accept opponent\'s stalemate offer',
+
+  inputs: {
+    currentState: {
+      type: 'ref',
+      descriptions: 'Object containing the current game state',
+      required: true,
+    },
+    /**
+     * @param { Object } requestedMove - Object describing the request for stalemate
+     * @param { MoveType.STALEMATE_ACCEPT } requestedMove.moveType - Specifies that this accepts a stalemate offer
+     * @param { 1 | 0 } requestedMove.playedBy - Which player is requesting a stalemate
+     */
+    requestedMove: {
+      type: 'ref',
+      description: 'Object containing data needed for current move',
+      required: true,
+    },
+    playedBy: {
+      type: 'number',
+      description: 'Player number of player requesting move',
+      required: true,
+    },
+    priorStates: {
+      type: 'ref',
+      description: "List of packed gameStateRows for this game's prior states",
+      required: true,
```

Comment:
Does this actually need to be required, since it seems it's only used in a few helpers

Reply:
So it's a little funky. The part that's weird is that the place that calls any one of these (currently only the `move` endpoint), has to be able to call _all_ of the `validate` and `execute()` helpers and to be able to do so by passing them the same input, since it grabs the 'right' functions based on the `moveType` and then no matter what they are, gives it the same arguments.

So it seemed more consistent to simply say it's required for all of them. We could make it optional for everything except the `stalemate-reject` stuff where this is actually needed (and I don't feel strongly about it), but I lean towards making them all the same signature for consistency's sake

Reply:
That's a fair point, I think the consistency is good.